apps/studio/src/lib/db/client.ts

```diff
@@ -165,6 +175,10 @@ export class DBConnection {
   getRoutineCreateScript = getRoutineCreateScript.bind(null, this.server, this.database)
   truncateAllTables = truncateAllTables.bind(null, this.server, this.database)
   setTableDescription = setTableDescription.bind(null, this.server, this.database)
+
+  // delete stuff
+  dropElement = dropElement.bind(null, this.server, this.database)
```

Comment:
Rather than writing a whole new method, you should use two helper methods I recently wrote - `bind` and bindAsync`, these are the first step to removing some of the duplication here:

![image](https://user-images.githubusercontent.com/279769/184981508-44332f08-3ff7-42a2-8c4e-510e1fd43e7b.png)